include/RAJA/policy/openmp/policy.hpp

```diff
@@ -288,6 +305,11 @@ struct omp_synchronize : make_policy_pattern_launch_t<Policy::openmp,
 ///////////////////////////////////////////////////////////////////////
 ///
 
+///
+/// Type alias for atomics
+///
+using policy::omp::omp_atomic;
```

Comment:
I think you might need to put the ifdef for MSVC here, instead of up in the namespace.